assets/admin/views/newsItemView.js

```diff
@@ -7,6 +7,11 @@ import Clipboard from "clipboard";
 
 export default class newsItemView {
   index() {
+    $(".modal-button").each(function () {
+      let modal_id_selector = $(this).attr("data-target");
+      let button_id = $(this).attr("id");
+      new Modal("#" + button_id, modal_id_selector);
```

Comment:
Prefer ES6 style string templates over concatenation where possible.

```suggestion
      new Modal(`#${button_id}${modal_id_selector}`);
```